metadata/src/test/java/edu/unc/lib/dl/util/DepositStatusFactoryTest.java

```diff
@@ -0,0 +1,38 @@
+package edu.unc.lib.dl.util;
+
+import static org.junit.Assert.*;
+import java.util.List;
+import java.util.Random;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import redis.clients.jedis.JedisPool;
+
+public class DepositStatusFactoryTest {
+	
+	private DepositStatusFactory factory;
+	private JedisPool jedisPool;
+	private final String uuid = Integer.toString(new Random().nextInt(99999));
```

Comment:
I know there is only one method currently, uuid and filenames should be local variables in the test method unless they are expected to be shared across test methods.  We would most likely want a different uuid for each test method anyways to make sure they don't interfere